xdp-dump/test-xdpdump.sh

```diff
@@ -0,0 +1,252 @@
+#
+# Test scrip to do basic xdpdump checks
+#
+# shellcheck disable=2039
+#
+XDPDUMP=./xdpdump
+XDP_LOADER=../xdp-loader/xdp-loader
+ALL_TESTS="test_help test_interfaces test_capt_pcap test_capt_term test_exitentry test_snap test_none_xdp"
+
+print_result()
+{
+    echo "$RESULT"
+    if [ -n "$1" ]; then
+	echo "ERROR: $1"
+    fi
+}
+
+start_background()
+{
+    local TMP_FILE="/tmp/${NS}_PID_$$_$RANDOM"
+    eval "${1} >& ${TMP_FILE} &"
+    local PID=$!
+    sleep 1 # Wait to make sure the command is executed in the background
+
+    mv "$TMP_FILE" "/tmp/${NS}_PID_${PID}" >& /dev/null
+
+    echo "$PID"
+}
+
+stop_background()
+{
+    local OUTPUT_FILE="/tmp/${NS}_PID_${1}"
+    kill -SIGINT "$1"
+    sleep 1 # Wait to make sure the buffer is flushed after the shutdown
+    cat "$OUTPUT_FILE"
+    rm "$OUTPUT_FILE" >& /dev/null
+}
+
+test_help()
+{
+    local RESULT=""
```

Comment:
This breaks with the expectation of a global variable above, doesn't it?

Reply:
Yes I forgot to cleanup this one :( RESULT should be global, will make the change